src/pages/Login/Login.js

```diff
@@ -1,8 +1,93 @@
 import React, { Component } from 'react';
+import { chkEmail, chkPwd } from '../../Validation/Validation';
+import '../Login/Login.scss';
 
 class Login extends Component {
+  state = {
+    email: '',
+    password: '',
+  };
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  goToMain = () => {
+    this.props.history.push('/main');
+  };
+
+  checkAll = e => {
+    const { email, password } = this.state;
+    if (chkEmail(email) && chkPwd(password) === true) {
```

Comment:
```suggestion
    if (chkEmail(email) && chkPwd(password)) {
```
chkEmail과 chkPwd가 리턴하는 boolean 값을 그대로 활용해도 되는 경우라서 === true는 생략해주세요

Reply:
> chkEmail과 chkPwd가 리턴하는 boolean 값을 그대로 활용해도 되는 경우라서 === true는 생략해주세요

=== true 제거 완료했습니다.